### ISSUE TITLE
Strategy to embed images

### ISSUE THREAD

**Issue by dcramer** [dcramer/gamegame#6]
There's a lot of visuals in rulebooks that'd be great to reference. Ignoring whether we can textual represent them, the `marker` parser we're using will actually transform text _and_ extract images at the same time. I believe we could design a system that lets us easily reference those in outputs (and embed them).

**Comment by tian-yi** [comment]
Yes, that's exactly copali for. Copali is used to embed the images. and direct semantically search the images using text. Also, the cool part is that you can then feed the retrieved image and text into a multi-modal LLMs like gpt-4o, gemini  etc. they will answer you questions using both.
